### ISSUE TITLE
Could you provide the fine-tuned weights?

### ISSUE THREAD

**Issue by CatfishW** [ZhangGongjie/Meta-DETR#72]
I finetuned your base-class weight on MS COCO dataset using RTX4070ti after 100 epochs with the batch size of 1 while other hyperparamter settings remain the same in accordance with the scripts you provided, but it does not reproduce to the accuracy to your essay out of the blue. The evaluation results for 10 shots are as follows:

Averaged stats: class_error: 76.92  loss: 25.7722 (22.3666)  loss_ce: 2.4055 (1.9647)  loss_bbox: 0.9593 (0.8582)  loss_giou: 1.2056 (1.0262)  loss_ce_0: 2.3303 (1.9005)  loss_bbox_0: 0.8101 (0.7742)  loss_giou_0: 1.0498 (1.0038)  loss_ce_1: 2.3550 (1.8903)  loss_bbox_1: 0.8683 (0.8341)  loss_giou_1: 1.0159 (0.9948)  loss_ce_2: 2.4136 (1.9047)  loss_bbox_2: 0.8322 (0.8132)  loss_giou_2: 1.0873 (0.9953)  loss_ce_3: 2.2909 (1.8730)  loss_bbox_3: 0.8844 (0.8044)  loss_giou_3: 1.0809 (1.0009)  loss_ce_4: 2.1820 (1.9072)  loss_bbox_4: 0.8562 (0.8144)  loss_giou_4: 1.0902 (1.0071)  loss_ce_unscaled: 1.2028 (0.9824)  class_error_unscaled: 50.0000 (43.5332)  loss_bbox_unscaled: 0.1919 (0.1716)  loss_giou_unscaled: 0.6028 (0.5131)  cardinality_error_unscaled: 299.5000 (299.4213)  loss_ce_0_unscaled: 1.1651 (0.9503)  loss_bbox_0_unscaled: 0.1620 (0.1548)  loss_giou_0_unscaled: 0.5249 (0.5019)  cardinality_error_0_unscaled: 298.4375 (298.3096)  loss_ce_1_unscaled: 1.1775 (0.9451)  loss_bbox_1_unscaled: 0.1737 (0.1668)  loss_giou_1_unscaled: 0.5080 (0.4974)  cardinality_error_1_unscaled: 299.3750 (299.0845)  loss_ce_2_unscaled: 1.2068 (0.9523)  loss_bbox_2_unscaled: 0.1664 (0.1626)  loss_giou_2_unscaled: 0.5437 (0.4976)  cardinality_error_2_unscaled: 298.7500 (298.6726)  loss_ce_3_unscaled: 1.1455 (0.9365)  loss_bbox_3_unscaled: 0.1769 (0.1609)  loss_giou_3_unscaled: 0.5405 (0.5004)  cardinality_error_3_unscaled: 299.3125 (299.0845)  loss_ce_4_unscaled: 1.0910 (0.9536)  loss_bbox_4_unscaled: 0.1712 (0.1629)  loss_giou_4_unscaled: 0.5451 (0.5035)  cardinality_error_4_unscaled: 299.4375 (299.2578)
 * Novel Categories:
Accumulating evaluation results...
DONE (t=1.25s).
IoU metric: bbox
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.075
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.163
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.061
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.015
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.061
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.127
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.141
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.213
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.222
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.043
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.160
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.360

Thus, I'm wondering if you could provide your fine-tuned weights, and my email address is 1819409756@qq.com.

Looking forward to your reply.

**Comment by nanfangAlan** [comment]
Are you finetuned with 8 x GPUs?
